.github/workflows/release.yml

```diff
@@ -16,28 +16,31 @@ jobs:
           YARN_NPM_AUTH_TOKEN: ${{ secrets.YARN_NPM_AUTH_TOKEN }}
 
   bump-ci-integrations:
+    name: Bump datadog-ci in integration
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         integration:
           - synthetics-ci-github-action
           - datadog-ci-azure-devops
           - synthetics-test-automation-circleci-orb
-
-    name: Bump datadog-ci in integration
-    runs-on: ubuntu-latest
-
     steps:
       - name: Create bump datadog-ci PR
         uses: actions/github-script@v6
         with:
           github-token: ${{ secrets.CROSS_REPOSITORY_GITHUB_TOKEN }}
           script: |
+            const tagName = '${{ github.event.release.tag_name }}'.replace('v', '')
```

Comment:
For more info on `github.event.release.tag_name`, see https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=released#release.

This line of code is similar to: https://github.com/DataDog/synthetics-ci-github-action/blob/93319f125c3ee84ad58e394c200a5d79927aa9ce/.github/workflows/release-version-on-merge.yml#L25